### ISSUE TITLE
V2 - Organize imports also organizes exports by default, Can this be a flag

### ISSUE THREAD

**Issue by skearney414** [biomejs/biome#6387]
### Environment information

```bash
CLI:
  Version:                      2.0.0
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         unset
  JS_RUNTIME_VERSION:           v23.6.1
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         bun/1.2.16

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         C:\Users\Sean.Kearney\Documents\Git\frontend\biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false

Linter:
  JavaScript enabled:           unset
  JSON enabled:                 unset
  CSS enabled:                  unset
  GraphQL enabled:              unset
  Recommended:                  false
  Enabled rules:
    complexity/noAdjacentSpacesInRegex
    complexity/noExtraBooleanCast
    complexity/noUselessCatch
    complexity/noUselessTypeConstraint
    correctness/noConstAssign
    correctness/noConstantCondition
    correctness/noEmptyCharacterClassInRegex
    correctness/noEmptyPattern
    correctness/noGlobalObjectCalls
    correctness/noInvalidBuiltinInstantiation
    correctness/noInvalidConstructorSuper
    correctness/noNonoctalDecimalEscape
    correctness/noPrecisionLoss
    correctness/noSelfAssign
    correctness/noSetterReturn
    correctness/noSwitchDeclarations
    correctness/noUndeclaredVariables
    correctness/noUnreachable
    correctness/noUnreachableSuper
    correctness/noUnsafeFinally
    correctness/noUnsafeOptionalChaining
    correctness/noUnusedImports
    correctness/noUnusedLabels
    correctness/noUnusedPrivateClassMembers
    correctness/noUnusedVariables
    correctness/useIsNan
    correctness/useValidForDirection
    correctness/useValidTypeof
    correctness/useYield
    style/noNamespace
    style/useAsConstAssertion
    suspicious/noAsyncPromiseExecutor
    suspicious/noCatchAssign
    suspicious/noClassAssign
    suspicious/noCompareNegZero
    suspicious/noControlCharactersInRegex
    suspicious/noDebugger
    suspicious/noDuplicateCase
    suspicious/noDuplicateClassMembers
    suspicious/noDuplicateObjectKeys
    suspicious/noDuplicateParameters
    suspicious/noExplicitAny
    suspicious/noExtraNonNullAssertion
    suspicious/noFallthroughSwitchClause
    suspicious/noFunctionAssign
    suspicious/noGlobalAssign
    suspicious/noImportAssign
    suspicious/noMisleadingCharacterClass
    suspicious/noMisleadingInstantiator
    suspicious/noPrototypeBuiltins
    suspicious/noRedeclare
    suspicious/noShadowRestrictedNames
    suspicious/noSparseArray
    suspicious/noUnsafeDeclarationMerging
    suspicious/noUnsafeNegation
    suspicious/noWith
    suspicious/useGetterReturn
    suspicious/useNamespaceKeyword

Workspace:
  Open Documents:               0
```

### Rule name

organizeImports

### Playground link

d

### Expected result

We should have the option to choose whether exports are ordered - this is a breaking change for us

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
